Cargo.toml

```diff
@@ -1,11 +1,14 @@
 [package]
```

Comment:
When linting the manifest, Clippy complained about missing metadata.  I added sane initial values, but I can also change them, if required. :wrench: 

Adding these metadata might also benefit your Clap implementation, see #27, since the crate parses the manifest.

Reply:
No, looks good to me!

Maybe, I have to admit, I do not know enough about clap, to know what it reads from the Cargo.toml.